components/BidsTable.tsx

```diff
@@ -97,7 +97,11 @@ export default function BidsTable(props: Props) {
               <td>{formatBalance(bid.walletBalance)}</td>
               <td>{lookup[bid.walletAddress]?.bids}</td>
               <td>{lookup[bid.walletAddress]?.wins || '0'}</td>
-              <td>
+              <td
+                style={{
+                  width: '20vw',
+                }}
+              >
```

Comment:
I'm assuming you are doing this because otherwise it changes the width of the column depending on the content? I don't like the `20vw` because `vw` means it's set up in a way that we'll have to test it on mobile and generally be aware of it.

My recommendation would be to always render the `clientId` icon, but if there's no client ID for that bid, we render it with `visibility: hidden`. This way it still takes the space, and the width of the content is going to be consistent.